src/main/java/com/example/petbuddybackend/service/user/ClientService.java

```diff
@@ -47,6 +55,34 @@ public void createClientIfNotExist(JwtAuthenticationToken token) {
         }
     }
 
+    public ClientDTO getClient(String clientEmail) {
+        Client client = getClientByEmail(clientEmail);
+        userService.renewProfilePicture(client.getAccountData());
+        return clientMapper.mapToClientDTO(client);
+    }
+
+    public ClientComplexInfoDTO addFollowingCaretakers(String clientEmail, Set<String> caretakerEmails) {
```

Comment:
Tak jak w kontrolerze. Myślę, że jakby dodać zamiast Set<String> po prostu String caretakerEmail to sie uprości